homeassistant/components/vegehub/config_flow.py

```diff
@@ -0,0 +1,176 @@
+"""Config flow for the VegeHub integration."""
+
+import logging
+from typing import Any
+
+from vegehub import VegeHub
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components.webhook import (
+    async_generate_id as webhook_generate_id,
+    async_generate_url as webhook_generate_url,
+)
+from homeassistant.config_entries import ConfigFlowResult
+from homeassistant.const import (
+    CONF_DEVICE,
+    CONF_HOST,
+    CONF_IP_ADDRESS,
+    CONF_MAC,
+    CONF_WEBHOOK_ID,
+)
+from homeassistant.helpers.service_info import zeroconf
+from homeassistant.util.network import is_ip_address
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class VegeHubConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for VegeHub integration."""
+
+    _hub: VegeHub
+
+    def __init__(self) -> None:
+        """Initialize the VegeHub config flow."""
+        self._hostname: str = ""
+        self.webhook_id: str = ""
```

Comment:
```suggestion
    _hostname: str
    webhook_id: str
```
should do the same and saves the init